### ISSUE TITLE
v2 docs out of sync with commands

### ISSUE THREAD

**Issue by mattfarina** [helm/helm#7667]
The `--devel` was added to `helm inspect` in v2.16. The current v2 docs don't reflect this.

**Comment by technosophos** [comment]
Should this be filed in the docs repo?

**Comment by bacongobbler** [comment]
The Helm v2 docs currently reside under the `docs` folder in the dev-v2 branch. It was not migrated over to the helm-www codebase. We still use the old process to build that site, so it seems fitting to keep it here

**Comment by bridgetkromhout** [comment]
Hi, @mattfarina - which part of the docs were you noticing was missing `--devel`? I see this line:
```
      --devel              Use development versions, too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored.
```

It's present on https://v2.helm.sh/docs/helm/#helm-inspect but perhaps another part of the v2 docs also needs it?

**Comment by bridgetkromhout** [comment]
(And @bacongobbler tells me there is a different place to look - sidenote, I think we should look into migrating anything we aren't auto-generating, if feasible, into the helm/helm-www repo.)

**Comment by bacongobbler** [comment]
> sidenote, I think we should look into migrating anything we aren't auto-generating, if feasible, into the helm/helm-www repo.)

The process was completed for Helm 3's documentation, but due to the nuances in how Helm 2's documentation is generated and assembled, unfortunately it's not that simple. I've tried and ran into several roadblocks. We came to a compromise instead: we'll continue maintaining the Helm 2 docs as it were, and maintain the Helm 3 docs on helm-www. This issue will disappear over time once we stop supporting Helm 2.

Here was the location I was referring to: https://github.com/helm/helm/blob/dev-v2/docs/using_helm.md
